pkg/storer/internal/chunkstore/transaction.go

```diff
@@ -165,16 +173,27 @@ func (cs *txChunkStoreWrapper) NewTx(state *storage.TxState) storage.TxChunkStor
 		toReleaseLocs: make(map[[32]byte]sharky.Location),
 		toReleaseSums: make(map[sharky.Location][32]byte),
 	}
+	chunkStore := &chunkStoreWrapper{
+		store:  cs.store,
+		sharky: txSharky,
+	}
 	return &txChunkStoreWrapper{
 		TxChunkStoreBase: &storage.TxChunkStoreBase{
-			TxState: state,
-			ChunkStore: &chunkStoreWrapper{
-				store:  cs.store,
-				sharky: txSharky,
-			},
+			TxState:    state,
+			ChunkStore: chunkStore,
 		},
 		store:    cs.store,
 		txSharky: txSharky,
+		revOps: storage.NewInMemTxRevertOpStore(
```

Comment:
The inmem revert op store might not be enough here right? I guess we need some sort of disk recovery mechanism here.